packages/amplify-cli/src/context-manager.ts

```diff
@@ -19,13 +19,12 @@ export async function attachUsageData(context: Context) {
   const usageTrackingEnabled = AMPLIFY_CLI_ENABLE_USAGE_DATA
     ? AMPLIFY_CLI_ENABLE_USAGE_DATA === 'true'
     : config.usageDataConfig.isUsageTrackingEnabled;
-  const accountId = await context.amplify.executeProviderUtils(context, 'awscloudformation', 'getAccountId');
   if (usageTrackingEnabled) {
     context.usageData = UsageData.Instance;
   } else {
     context.usageData = NoUsageData.Instance;
   }
-  context.usageData.init(config.usageDataConfig.installationUuid, getVersion(context), context.input, accountId, getProjectSettings());
+  context.usageData.init(config.usageDataConfig.installationUuid, getVersion(context), context.input, '', getProjectSettings());
```

Comment:
Will we need that empty string argument later?

Reply:
I will need it or it will break validations on the back end